x-pack/plugins/synthetics/public/apps/synthetics/components/getting_started/getting_started_page.tsx

```diff
@@ -5,54 +5,152 @@
  * 2.0.
  */
 
-import React, { useEffect } from 'react';
-import { EuiEmptyPrompt, EuiLink, EuiSpacer, EuiText } from '@elastic/eui';
-import { useDispatch } from 'react-redux';
+import React, { useEffect, useCallback } from 'react';
+import {
+  EuiEmptyPrompt,
+  EuiLink,
+  EuiSpacer,
+  EuiText,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { useDispatch, useSelector } from 'react-redux';
 import { useHistory } from 'react-router-dom';
 import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
 import styled from 'styled-components';
-import { useBreadcrumbs } from '../../hooks';
-import { getServiceLocations } from '../../state';
+import { useBreadcrumbs, useLocations, useFleetPermissions } from '../../hooks';
+import { usePrivateLocationsAPI } from '../settings/private_locations/hooks/use_locations_api';
+import { LoadingState } from '../monitors_page/overview/overview/monitor_detail_flyout';
+import {
+  getServiceLocations,
+  selectAddingNewPrivateLocation,
+  setAddingNewPrivateLocation,
+  getAgentPoliciesAction,
+  selectAgentPolicies,
+} from '../../state';
 import { MONITOR_ADD_ROUTE } from '../../../../../common/constants/ui';
+import { PrivateLocation } from '../../../../../common/runtime_types';
 import { SimpleMonitorForm } from './simple_monitor_form';
+import { AddLocationFlyout } from '../settings/private_locations/add_location_flyout';
 
 export const GettingStartedPage = () => {
   const dispatch = useDispatch();
   const history = useHistory();
 
+  const { canReadAgentPolicies } = useFleetPermissions();
+
   useEffect(() => {
     dispatch(getServiceLocations());
-  }, [dispatch]);
+    if (canReadAgentPolicies) {
+      dispatch(getAgentPoliciesAction.get());
+    }
+  }, [canReadAgentPolicies, dispatch]);
 
   useBreadcrumbs([{ text: MONITORING_OVERVIEW_LABEL }]); // No extra breadcrumbs on overview
 
-  return (
+  const { locations, loading: allLocationsLoading } = useLocations();
+  const { loading: agentPoliciesLoading } = useSelector(selectAgentPolicies);
+  const loading = allLocationsLoading || agentPoliciesLoading;
+
+  const hasNoLocations = !allLocationsLoading && locations.length === 0;
+
+  return !loading ? (
     <Wrapper>
+      {hasNoLocations ? (
+        <GettingStartedOnPrem />
+      ) : (
+        <EuiEmptyPrompt
+          title={<h2>{CREATE_SINGLE_PAGE_LABEL}</h2>}
+          layout="horizontal"
+          color="plain"
+          body={
+            <>
+              <EuiText size="s">
+                {OR_LABEL}{' '}
+                <EuiLink
+                  href={history.createHref({
+                    pathname: MONITOR_ADD_ROUTE,
+                  })}
+                >
+                  {SELECT_DIFFERENT_MONITOR}
+                </EuiLink>
+                {i18n.translate('xpack.synthetics.gettingStarted.createSingle.description', {
+                  defaultMessage: ' to get started with Elastic Synthetics Monitoring',
```

Comment:
```suggestion
                  defaultMessage: ' to get started with Elastic Synthetics Monitoring.',
```
Punctuation

Note: we're also missing ending punctuation on help text here under the 2 fields.
<img width="422" alt="image" src="https://user-images.githubusercontent.com/10208282/223675111-ab732df5-a6fa-4b6f-8fcb-30b07d2e95aa.png">